count_server/web/router.ex

```diff
@@ -16,11 +16,17 @@ defmodule CountServer.Router do
   scope "/", CountServer do
     pipe_through :browser # Use the default browser stack
 
-    get "/", CounterController, :index
+    # get "/", CounterController, :index
+    get "/", HomeController, :index
   end
 
   # Other scopes may use custom stacks.
-  # scope "/api", CountServer do
-  #   pipe_through :api
-  # end
+  scope "/api/v1", CountServer do
```

Comment:
hmm.  Well don't think I'll need a sessionController for the browser pipeline as I'm building my front end in React, but really good point.  I don't know either.